Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+controls payments, see [Mining setup](Mining-Setup "wikilink"). You must
+have over 100 RAC on at least 1 whitelisted BOINC project before you can
+send a beacon to receive Research credit. After your CPID is in the
+Neural Network, all research credits you earn will accrue in a "savings
+account" for a rolling 14 day period. You have up to 2 full weeks to
+stake a block without losing any research credit
+earned.
+
+## Is it true that someone would not lose reward if blocks are created on a fork?
+
+YES! If you find a block on a fork your "owed" balance will be reduced
+on that fork but it won't be reduced on the main block chain because
+that block wasn't really found by you on that fork. So once you sync
+back up with the main chain you will see the block you found change to
+an orphan (with the ?) and the balance in your research savings account
+will increase by that amount so that when you find a block on the main
+chain you will be paid for the research you are owed. This is what makes
+this method so much more fair, than the old PoW system. With
+Proof-of-Research as long as you find enough blocks in a 2 week window
+to be paid your full reward within the payout rules you will receive all
+the rewards you earned in the same period.
+
+## What should I put in my config-file for Gridcoin-Research?
+
+Please find gridcoinresearch.conf in your
+%appdata%\\roaming\\gridcoinresearch folder and insert a line for your
+BOINC email address if it is empty:
+
+`email=YourBoincEmail`  
+`addnode=node.gridcoin.us`
+
+These lines should already suffice. Else, find more at [GridcoinResearch
+Config
+file](Gridcoinresearch-Config-File "wikilink")
+
+## How can I tell if I've mined a block?
+
+First look at the transaction list for "PoR" payments.
+
+If you don't have any PoR yet, do a "list mymagnitude" in the debug
+console and look for the savings account balance "owed". That will be
+what you accrue for your first payment. It has to be higher than the
+subsidyschedule max block / 10, IE 40 right now in order to actually
+generate.
+
+## Reward calculation: what is RAC?
+
+<https://en.wikipedia.org/wiki/BOINC_Credit_System#Recent_average_credit>
+
+## Reward calculation: what is magnitude?
+
+The reward calculation is based on the relative contribution of a user
+to the total contribution of team Gridcoin. This is being calculated
+within every project and the sum of the relative contributions is the
+magnitude which is being used to calculate the reward. Besides the
+research rewards, each mined block contains interest.
+
+## Is it possible to join a pool?
+
+Yes, there are pools at <https://grcpool.com/> and
+<https://grc.arikado.ru/>
+
+## I am BOINCing nonstop but my magnitude is very low. What can I do?
+
+Regarding your magnitude: You should have an eye on your RAC and the
+project total RAC within team Gridcoin. Be aware that BOINC RAC is
+measured weekly, with the most recent week being higher weighted, the

```

Comment:
This part is confusingly worded. Maybe say `RAC is a rolling average over the past month. It is weighted to increase the value of more recent contributions`